OSDOC#43771 - Updates to NAD CLI configuration (#98488)
openshift/openshift-docs: modules/virt-creating-localnet-nad-cli.adoc

            "topology": "localnet", <4>
            "physicalNetworkName": "localnet-network", <5>
            "mtu": 1500, <6>
            "vlanID": 200, <7>
            "netAttachDefName": "default/localnet-network" <8>
    }
----
<1> The CNI specification version. The required value is `0.3.1`.
<2> The name of the network.
<3> The name of the CNI plug-in to be configured. The required value is `ovn-k8s-cni-overlay`.
<4> The topological configuration for the network. The required value is `localnet`.
<5> The name of the host's physical network interface, to which the pod's new network interface will be attached. This setting can be reused by multiple NADs. Note that `physicalNetworkName` must match the value of the `spec.desiredState.ovn.bridge-mappings.localnet` field of the `NodeNetworkConfigurationPolicy` object that defines the OVS bridge mapping.
<6> Optional: The maximum transmission unit (MTU) value. If you do not set a value, the Cluster Network Operator (CNO) sets a default MTU value by calculating the difference among the underlay MTU of the primary network interface, the overlay MTU of the pod network, and byte capacity of any enabled features, such as IPsec.

> <7> Optionally, for more fine-grained network management, you can configure a virtual LAN (VLAN) ID for the NAD. VMs that use this NAD have an interface that can communicate only with devices that use the same VLAN ID (`200` in this example).

Reviewer: ```suggestion
<7> Optional: The virtual LAN (VLAN) ID for the NAD. VMs that use this NAD have an interface that can communicate only with devices that use the same VLAN ID. In this example, the VLAN ID is `200`.
```
